nfv_ovs_dpdk.md

```diff
@@ -0,0 +1,265 @@
+# NFV OVS DPDK
+
+This document describes how to deploy EDPM nodes with OVS DPDK.
+
+## Procedure Overview
+
+1. Configure kernel args and tuned parameters
+2. Configure OVS DPDK parameters
+3. Configure the networks with DPDK interfaces of the EDPM nodes
+2. Configure Nova and Neutron parameters.
+
+In order to complete the above procedure, the `services` list of the
+`OpenStackDataPlaneNodeSet` CR needs to be edited.
+
+## OpenStackDataPlaneNodeSet services list
+
+EDPM nodes can be configured by creating an
+`OpenStackDataPlaneNodeSet` CR which the
+[dataplane-operator](https://openstack-k8s-operators.github.io/dataplane-operator)
+will reconcile to create OpenStackDataPlaneService resources
+when an `OpenStackDataPlaneDeployment` CR is created.
+These types of CRs have a `services` list like the following:
+
+```yaml
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+spec:
+  ...
+  services:
+    - bootstrap
+    - download-cache
+    - reboot-os
+    - configure-ovs-dpdk
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - ssh-known-hosts
+    - run-os
+    - install-certs
+    - ovn
+    - neutron-ovn
+    - neutron-metadata
+    - libvirt
+    - nova-custom-ovsdpdk
+```
+Only the services which are on the list will be configured.
+
+## Configure the EDPM ansible variables of the EDPM nodes
+
+Following are the list of EDPM ansible variables which need to be
+provided for deploying with OVS DPDK support.
```

Comment:
> Link to roles involved in the setup would provide more up to date information about variables involved. Especially since the roles may change in the future and they may also contain other information about nature of "services" deployed.

done